docker-compose.yml

```diff
@@ -2,17 +2,36 @@ version: '3.1'
 services:
   court-register:
     image: quay.io/hmpps/court-register:latest
-    networks:
-      - hmpps
+    network_mode: "host"
```

Comment:
for some reason I thought that didn't work on a Mac ....

Reply:
I copied from PTPU - I've not tested that yet.